parlai/scripts/torchscript.py

```diff
@@ -96,7 +96,7 @@ def setup_args() -> ParlaiParser:
     )
     parser.add_argument(
         "-eio",
-        "--enable-inference-optimization",
+        "--enable_inference_optimization",
```

Comment:
but this should be dashes :P

Reply:
I can make it use dashes for consistency with the other parameters. Let me know which one you prefer